iroh-blobs/src/store/mem.rs

```diff
@@ -43,20 +43,22 @@ pub struct Store {
 #[derive(Debug, Default)]
 struct StoreInner(RwLock<StateInner>);
 
-impl LivenessTracker for StoreInner {
-    fn on_clone(&self, inner: &HashAndFormat) {
-        tracing::trace!("temp tagging: {:?}", inner);
-        let mut state = self.0.write().unwrap();
-        state.temp.inc(inner);
-    }
-
+impl TagDrop for StoreInner {
```

Comment:
why did you make two traits, is there a use case for only implementing one of the two?

Reply:
The only trait that most people will interact with is TagDrop. The other one is just for store implementers.

I don't want to give the ability to increase the reference count to temp tags. In case of the rpc there isn't even message for that...